spec/requests/v1/songs_request_spec.rb

```diff
@@ -11,3 +11,19 @@
     end
   end
 end
+
+describe 'song endpoint' do
+  describe 'GET /song' do
+    it 'goes to the correct endpoint' do
+      assert_generates '/songs/1', { controller: 'v1/songs', action: 'show', id: '1' }
+    end
```

Comment:
We don't usually bother checking that passing those params generates the right URL - we kind of trust Rails to do that part for us.  Plus our other request tests should break if that *didn't* work.  Nice finding `assert_generates`, though!